tests/test_style/test_rcmod.py

```diff
@@ -38,9 +38,25 @@ class RCParamTester(VisualTestCase):
     """
 
     excluded_params = {
-        "backend",  # This cannot be changed by manipulating rc
+        "backend",  # This cannot be changed by manipulating rc,
```

Comment:
gah, how did that stray comma get there? is it worth it to run CI again just to get rid of it?